101-aci-dynamicsnav/azuredeploy.json

```diff
@@ -60,7 +60,7 @@
       {
               "name": "[parameters('name')]",
               "type": "Microsoft.ContainerInstance/containerGroups",
-              "apiVersion": "2017-10-01-preview",
+              "apiVersion": "2016-05-15",
```

Comment:
this is not a valid apiVersion for this RP...  I think you can just revert this change